api/qanpb/json/client/object_details/get_metrics_responses.go

```diff
@@ -531,6 +538,25 @@ func (o *GetMetricsOKBody) validateTotals(formats strfmt.Registry) error {
 	return nil
 }
 
+func (o *GetMetricsOKBody) validateMetadata(formats strfmt.Registry) error {
+	if swag.IsZero(o.Metadata) { // not required
+		return nil
+	}
+
+	if o.Metadata != nil {
```

Comment:
swag.IsZero should cover nil check, this check is redundant

Reply:
I agree with you but did not edit it.
// Code generated by go-swagger; DO NOT EDIT.